ci/docker/android-builder/packages.txt

```diff
@@ -0,0 +1,6 @@
+build-tools;30.0.3
```

Comment:
Do we have plans to migrate to 30 in this quarter?

Reply:
no i don't think so, 
but we could try to use higher build-tools version for building 29 api